src/plugins/es_ui_shared/public/components/view_api_request_flyout/view_api_request_flyout.tsx

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+// We want to allow both right-clicking to open in a new tab and clicking through
+// the "Open in Console" link. We could use `RedirectAppLinks` at the top level
+// but that inserts a div which messes up the layout of the flyout.
+/* eslint-disable @elastic/eui/href-or-on-click */
```

Comment:
I havent tried that, but it does look like it could work! I'm a bit thorn about implementing the `RedirectAppLinks` though, because it requires an `ApplicationStart` instance and it would mean we will have to add another required dependency to the flyout for it and have to retrieve it from consumers as well. What do you think?

Reply:
I think actually we could reduce the number of required dependencies for the flyout, because currently we need to pass both props `navigateToUrl` and `canShowDevtools` which are derived from `ApplicationStart`. So instead we could pass `ApplicationStart` as a whole and the flyout can get `navigateToUrl` and compute `canShowDevtools` from it. That would also hide implementation details from consumers, as they would not need to compute if dev tools console is available.

Reply:
Good point! I didnt see that they both come from the same place, I've refactored it to only consume ApplicationStart instead. 

PS: I did have to add `kibanaReact` as a dependency of `es_ui_shared` since thats where `RedirectAppLinks` lives